src/test/java/org/csanchez/jenkins/plugins/kubernetes/KubernetesTestUtil.java

```diff
@@ -142,9 +142,39 @@ public static void assumeKubernetes() throws Exception {
         return l;
     }
 
+    public static Map<String, String> getAnnotations(KubernetesCloud cloud, String podName) {
```

Comment:
you are not using this method

Reply:
It's being used in KubernetesPipelineTest and KubernetsDeclarativeAgent test - 
![image](https://user-images.githubusercontent.com/340093/53030653-dc438d80-3430-11e9-8565-87dff002b8a8.png)

Reply:
yes, you are using it in the tests but not in the actual plugin, there's nothing that will add the annotations to the pods when they are launched

Reply:
The annotation is being added automatically by the step execution - https://github.com/jenkinsci/kubernetes-plugin/pull/433/files#diff-5501a99a43da60cbad908396dfee9336R100 - the method is necessary to confirm that the the annotation was added and properly reflects the build url.

Reply:
my bad, missread it